### PR TITLE
Teradata dbt utils 1.3.0

### PR DESCRIPTION
**Pull request by septimit**
### Description
Update teradata-utils to dbt-utils 1.3.0
Fixes at_least_one macro
Decommissions the old test suite.

### Checklist
 - [x] I have run this code in development and it appears to resolve the stated issue
 - [ ] This PR includes tests, or tests are not required/relevant for this PR
 - [x] I have updated the `CHANGELOG.md` with information about my change

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## teradata_utils 1.3.0
+
```
Comment (VarunSharma15) [inline CHANGELOG.md:2]: We need to add insert_by_period materialization as a feature in this changelog.md file Write Preview Cancel Update comment

File: setup.py [hunk 2]
```diff
@@ -1,60 +0,0 @@
-#!/usr/bin/env python
```
Comment (VarunSharma15) [inline setup.py:1]: @septimit It's a GitHub release and we do not require this file. Is that the reason for deleting this file?

File: requirements_dev.txt [hunk 3]
```diff
@@ -1,9 +0,0 @@
-twine
```
Comment (VarunSharma15) [inline requirements_dev.txt:1]: @septimit We are testing this package with dbt-utils tests, is this the reason for removing this file ?

Comment (septimit) [reply requirements_dev.txt:1]: Correct, I propose we only use the official tests from dbt-utils and thus removing the custom test suite. This file does not need to be here anymore.

File: setup.py (hunk 2 above)
Comment (septimit) [reply setup.py:1]: I propose we only use the official tests from dbt-utils and thus removing the custom test suite. This file does not need to be here anymore.

**Review by tallamohan — APPROVED** [review]
Looks Good

**Review by VarunSharma15 — APPROVED** [review]

**Pull request merged**